package.json

```diff
@@ -14,7 +14,7 @@
     "build": "yarn preflight && next build",
     "update-and-build": "yarn git-update && yarn build && yarn generate-sitemap",
     "start": "next start",
-    "test": "loadr -- uvu",
+    "test": "npx playwright test",
```

Comment:
`playwright test` since it's now a dev dependency.

Reply:
Updated